saleor/order/utils.py

```diff
@@ -138,7 +138,7 @@ def update_order_prices(order, discounts):
 
     if order.shipping_method:
         order.shipping_price = manager.calculate_order_shipping(order)
-        order.save()
+        order.save(update_fields=["shipping_price"])
```

Comment:
Same for `shipping_price`:
```suggestion
        order.save(update_fields=["shipping_price_net_amount", "shipping_price_gross_amount", "currency"])
```